generic/README.md

```diff
@@ -85,6 +85,11 @@
  ([source](https://raw.githubusercontent.com/MurdoMaclachlan/ToR-Repost-Collection/master/generic/hard-to-swallow-pills/hard-to-swallow-pills.md)) 
  ([preview](hard-to-swallow-pills/hard-to-swallow-pills.md)) 
  ([image](hard-to-swallow-pills/hard-to-swallow-pills.png))
+ 
+**I Am Once Again Asking For Your Financial Support**: A meme which is a screenshot of Bernie Sanders from an ad, and text.
+ ([source](https://raw.githubusercontent.com/Ryan-Huang1/ToR-Repost-Collection/master/generic/i-am-once-again-asking-for-your-financial-support/i-am-once-again-asking-for-your-financial-support.md)) 
```

Comment:
Could you change the "Ryan-Huang1" part of this link to "MurdoMaclachlan"? Otherwise the link will break on being merged :sweat_smile: